chainer/computational_graph.py

```diff
@@ -59,18 +67,35 @@ class ComputationalGraph(object):
         rankdir (str): Direction of the graph that must be
             TB (top to bottom), BT (bottom to top), LR (left to right)
             or RL (right to left).
+        remove_variable (bool): If `True`, :class:`~chainer.Variable`s are
+            removed from the resulting computational graph. Only
+            :class:`~chainer.Function`s are shown in the output.
+        show_name (bool): If `True`, the `name` attribute of each node is
+            added to the label of the node. Default is `True`.
+
+    .. note::
+
+        To make the same graph as the previous versions, specify
+        `variable_style=None`, `function_style=None`, and `show_name=False`
+        explicitly. The previous versions than `v1.21.0` outputs the same
+        results as when it takes these arguments, but after `v1.22.0`, the
+        output has the richest representation as default, namely, styles are
+        set and names are shown.
```

Comment:
> To make the same graph as the previous versions
"the previous versions" is not clear in this sentence. I think it is better to change the order of explanations so that we first explain the default behavior has changed, then show how to reproduce the previous results.

> The previous versions than `v1.21.0`
This sounds like v1.21.0 is not included, is it correct?

And we need to fix the versions because this PR would be merged as v1.23.0.